extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/HttpConfiguration.java

```diff
@@ -91,6 +91,12 @@ public class HttpConfiguration {
      */
     public ServerSslConfig ssl;
 
+    /**
+     * If this is true vertx will handle 100-continue requests
+     */
+    @ConfigItem(defaultValue = "false", name = "handle-100-continue-automatically")
+    public boolean handle100ContinueAutomatically;
```

Comment:
Not a fan of this very long name to be honest :)

Perhaps @gastaldi has a better idea

Reply:
My goal was to keep the naming as close as possible to the naming in vert.x. I hope that people will find the vert.x documentation and dicussions around this feature toggle more easily this way...

Reply:
I doubt that people will find the feature from the Vert.x documentation - it will more likely be found by our configuration page

Reply:
Yeah not a big fan of the long name too, but I don't have any suggestions here unfortunately :(

Reply:
I think we can live with the long name given it's an advanced option. At least it's descriptive and says what it does.

Reply:
If I were to suggest something, I'd point that `100` and `continue` are the same thing, so I'd remove the `100` from the property name ;)

Reply:
Except `100-continue` makes it clear that it's an HTTP status whereas just `continue` is not as clear. People will copy/paste it anyway so I think clarity is better than brevity.